openshift_scalability/PrometheusLoader.md

```diff
@@ -0,0 +1,15 @@
+# Intro
+this tools used to fire prometheus queries against prometheus instance in OpenShift
```

Comment:
Editorial:  "This tool is used to fire prometheus queries against a prometheus instance in OpenShift."